buildSrc/src/main/kotlin/com.hedera.block.jpms-modules.gradle.kts

```diff
@@ -52,6 +52,11 @@ jvmDependencyConflicts.patch {
     }
     module("io.grpc:grpc-stub") { annotationLibraries.forEach { removeDependency(it) } }
     module("io.grpc:grpc-util") { annotationLibraries.forEach { removeDependency(it) } }
+    // Added for metrics and logging, but also several platform classes
+    module("io.prometheus:simpleclient") {
+        removeDependency("io.prometheus:simpleclient_tracer_otel")
+        removeDependency("io.prometheus:simpleclient_tracer_otel_agent")
+    }
```

Comment:
I think I cannot do the change since the `swirlds.common` module needs it.
see here: 
https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-common/src/main/java/module-info.java#L154
https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-common/src/main/java/module-info.java#L159

Maybe we can open up a ticket for `platform` team to do the change there and then we can do the change once they have do it.

Good that we have several months before the first production release.

Reply:
We might need to help platform by making the upgrade for them.  It's not tiny, but should be very doable.

Reply:
https://github.com/hashgraph/hedera-services/issues/14696

Marking as resolved, created a ticket for services repo, we might consider when we help do it.